### PR TITLE
index.html: Add support for markdown

### PR DESCRIPTION
**Pull request by hemangsk**
Closes https://github.com/coala/projects/issues/8

screen shot 2017-01-18 at 12 50 47 am

- wrt fee8f34
Eg: The link in 3rd line -  'the linter documentation' is being converted to HTML
screen shot 2017-01-18 at 3 21 28 pm

### CODE REVIEW

File: partials/tabs/projects.html [hunk 1]
```diff
@@ -30,46 +30,44 @@
...
         <div ng-show="currentProject.requirements.length>0" class="project-detail-element  reqs-proj">
           <div class="small-heading">REQUIREMENTS</div>
-          <li class="pr-element-detail milestones" ng-repeat="req in currentProject.requirements">{{ req }}</li><br>
+          <li btf-markdown="req" class="pr-element-detail milestones markdown" ng-repeat="req in currentProject.requirements"></li><br>
         </div>
 
         <div ng-show="currentProject.initiatives.length>0" class="project-detail-element">
           <div class="small-heading">INITIATIVES</div>
-          <span class="pr-element-detail chip" ng-repeat="initiative in currentProject.initiatives">{{ initiative }}</span><br>
+          <span btf-markdown="initiative" class="pr-element-detail chip markdown" ng-repeat="initiative in currentProject.initiatives"></span><br>
```
Comment (sils) [inline partials/tabs/projects.html]: I don't think we need markdown for the initiatives, tags and other one-word things

File: partials/tabs/projects.html [hunk 2]
```diff
@@ -30,46 +30,44 @@
...
       <div class="col-md-8 dashboard">
-        <div class="project-description">
-          {{currentProject.desc}}
+        <div btf-markdown="currentProject.desc" class="project-description markdown">
         </div>
         <div class="row milestones-row">
           <div class="milestones medium-heading">MILESTONES</div>
           <div ng-repeat="milestone_header in currentMilestone">
-           <div class="small-heading">{{milestone_header}}</div>
+           <div btf-markdown="milestone_header" class="small-heading markdown"></div>
```
Comment (sils) [inline partials/tabs/projects.html]: this is also more of a term, the milestone descriptions may have markdown but not the names, they're not real text, just identifiers

Comment (hemangsk) [inline partials/tabs/projects.html]: gotcha. improving it

**Comment by hemangsk** [comment]
21a91aa is  a quick proof of concept for #51 

screen shot 2017-01-18 at 7 57 03 pm

**Comment by hemangsk** [comment]
Ported Project 1 to Markdown in 137ea37

screen shot 2017-01-18 at 9 55 42 pm

File: data/projects/improve_diff_handling.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Bears should be able to offer more than just one possible patch 
+for an issue. Users should be able to choose between different patches, e.g. 
+for different possible spelling corrections.
+
+
```
Comment (sils) [inline data/projects/improve_diff_handling.md]: two newlines are sufficient

File: resources/css/style.css [hunk 4]
```diff
@@ -253,6 +256,18 @@ a:hover > .card{
...
+}
+.project-description > p > a:hover{
+  opacity: 0.7;
+}
+.milestone > p > a{
+  color: azure;
+}
+.milestone > p > a:hover{
+  opacity: 0.7;
+}
```
Comment (sils) [inline resources/css/style.css]: shouldn't you remove like all milestone rlated css?

File: data/projects/create_jenkins_plugin.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+Jenkins users should be able to see coala results right in the 
+Jenkins UI. The result of this project should be a full fledged Jenkins 
+integration, living in an own repository with a full testsuite.
+
+If you find this fun you'll be the one knowing this component best and will 
+get the chance to maintain it on your own if you want - always with the
+support of the community guarding your back of course.
+
+#### MILESTONES
```
Comment (sils) [inline data/projects/create_jenkins_plugin.md]: why upper case? Let your CSS do that please

File: partials/tabs/projects.html [hunk 6]
```diff
@@ -96,7 +87,7 @@
       opacity: 0.8, // Opacity of modal background
       in_duration: 300, // Transition in duration
       out_duration: 200, // Transition out duration
-      starting_top: '50%', // Starting top style attribute
+      starting_top: '10%', // Starting top style attribute
```
Comment (sils) [inline partials/tabs/projects.html]: why this change?

Comment (hemangsk) [inline partials/tabs/projects.html]: linter integration project's popup was very large and was not fitting in the window, therefore this shifts the box a little up.

Before
screen shot 2017-01-19 at 1 28 23 am
After

screen shot 2017-01-19 at 1 28 07 am

Comment (sils) [inline partials/tabs/projects.html]: I see ok

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/59/commits/911b1f1fa48654653de13dba652163c11090cce8

**Pull request merged**